ansible/roles/gateone-gen-sshkey/tasks/main.yml

```diff
@@ -30,5 +29,30 @@
   delegate_to: gateone
 
 - name: finally, transfer the public key locally
-  fetch: src="/var/lib/gateone/users/{{ ATMOUSERNAME }}/.ssh/{{ ATMOUSERNAME }}_default.pub" dest="{{ GATEONE_LOCAL_PUBKEY_DIR | default('/tmp/') }}" flat=yes
+  fetch: src= "/var/lib/gateone/users/{{ ATMOUSERNAME }}/.ssh/{{ ATMOUSERNAME }}_default.pub" dest= "{{ GATEONE_LOCAL_PUBKEY_DIR | default('/tmp/') }}" flat=yes
```

Comment:
This line executes fine//is syntactically correct.. I added the space was added because `vim`'s parser goes ape- ⁉️ when you `="`

The other option would be to nest the values instead of inline?